app/messaging/index.js

```diff
@@ -1,7 +1,23 @@
-const { createMessage } = require('./create-message')
-const { sendMessages } = require('./send-messages')
+const { MessageReceiver } = require('ffc-messaging')
+const { messagingConfig } = require('../config')
+const processDemographicsMessage = require('./process-demographics-message')
+const updateReceivers = []
 
-module.exports = {
-  createMessage,
-  sendMessages
+const start = async () => {
+  for (let i = 0; i < messagingConfig.updatesSubscription.numberOfReceivers; i++) {
+    let updateReceiver  // eslint-disable-line
+    const updateAction = message => processDemographicsMessage(message, updateReceiver)
+    updateReceiver = new MessageReceiver(messagingConfig.updatesSubscription, updateAction)
+    updateReceivers.push(updateReceiver)
+    await updateReceiver.subscribe()
+    console.info(`Receiver ${i + 1} ready to receive payment requests`)
+  }
```

Comment:
I'd suggest changing this so there is only ever one receiver.  Otherwise we may process updates out of sequence downstream, if one overtakes the other.